app/helpers/uuid_helper.py

```diff
@@ -0,0 +1,9 @@
+from uuid import UUID
```

Comment:
from being in the helpers module we know this is a helper, so no need to add the `_helper` suffix to the module name

Reply:
I agree 100% but I kept it to be consistent our existing ones.